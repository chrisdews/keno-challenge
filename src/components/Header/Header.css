.header {
    font-size: 2rem;
    color: white;
    background-color: rgb(226, 84, 84);
    margin: 10px;
    border-radius: 3px;
}