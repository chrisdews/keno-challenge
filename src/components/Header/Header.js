import React from "react";
import PropTypes from "prop-types";
import './Header.css'

const Header = ({ children }) => {
  return <div className="header" data-testid='keno-header'>{children}</div>;
};

Header.propTypes = {
  children: PropTypes.any,
};

export default Header;
