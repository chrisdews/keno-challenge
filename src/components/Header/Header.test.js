import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';
import { render, screen } from '@testing-library/react'

describe ('Header', ()=>{
    it('renders correctly - snapshot', () => {
        const tree = renderer
          .create(<Header><p>Keno</p></Header>)
          .toJSON();
        expect(tree).toMatchSnapshot();
      });

    test('renders Children', () => {
        render(<Header><p>Keno</p></Header>)
        expect(screen.getByText('Keno'))
    })
})

