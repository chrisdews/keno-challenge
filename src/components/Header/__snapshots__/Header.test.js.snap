// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly - snapshot 1`] = `
<div
  className="header"
  data-testid="keno-header"
>
  <p>
    Keno
  </p>
</div>
`;
